Recruit-Training/Embedded-Recruit-Training/.project

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```

Comment:
I was planning on leaving the option open. Cause the way its setup rn you can run make on it or use the IDE. This file eases the project import process, so I say we leave it since it's just a single file.

Reply:
Sounds good